### PR TITLE
Added colorspace to 2d context attributes

### LINKED ISSUE #15995: Missing colorSpace attribute in HTMLCanvasElement.getContext()
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext

### What specific section or headline is this issue about?

Syntax > Parameters > `contextAttributes`

### What information was incorrect, unhelpful, or incomplete?

There should be `colorSpace` under "2d context attributes".

### What did you expect to see?

- **`colorSpace`**: Specifies the color space of the rendering context. (quoted from the spec)

### Do you have any supporting links, references, or citations?

https://html.spec.whatwg.org/multipage/canvas.html#canvasrenderingcontext2dsettings

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/htmlcanvaselement/getcontext`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/htmlcanvaselement/getcontext/index.md
* Last commit: https://github.com/mdn/content/commit/7ca488eb307442881e796d125e432ec8ecdb7a5e
* Document last modified: 2022-05-10T07:08:54.000Z

</details>

### PR DESCRIPTION
**Pull request by dawei-wang**
Added colorSpace to 2d context attributes 
Fixes mdn/content#15995

### CODE REVIEW

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @dawei-wang !

I think it would be good to list possible values for this attribute, as we do in this page, for example, for `contextType`. From https://html.spec.whatwg.org/multipage/canvas.html#predefinedcolorspace, it looks like the values are "srgb" or "display-p3".

- The list of possible values can just be an unordered list and values should be quoted, as we do for `contextType` in this page.
- It looks like this is an optional attribute and "srgb" is the default. We should indicate optionality using the `{{optional_inline}}` macro call, as we do for instance for `contextAttributes` in this page. We should note next to the item for "srgb" that it is the default.
- It would be helpful to have a link to more detail on the color spaces. Unfortunately we don't seem to have resources for this on MDN, but maybe https://en.wikipedia.org/wiki/SRGB and https://en.wikipedia.org/wiki/DCI-P3 would do?

**Comment by dawei-wang** [comment]
I have added possible values, indicated that this is an optional attribute, and links to more detail on the color spaces. I think that this is ready to be merged.

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 1]
```diff
@@ -76,6 +75,10 @@ getContext(contextType, contextAttributes)
         contains an alpha channel. If set to `false`, the browser now knows
         that the backdrop is always opaque, which can speed up drawing of transparent
         content and images.
+    - `colorSpace` {{optional_inline}}
+      - : Specifies the color space of the rendering context. Possible values are:
+        - `"srgb"` - default value
+        - `"display-p3"`
```
Comment (wbamberg) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:81]:
```suggestion
        - `"srgb"` selects the [sRGB color space](https://en.wikipedia.org/wiki/SRGB). This is the default value.
        - `"display-p3"` selects the [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3).
```

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 2]
```diff
@@ -189,3 +192,4 @@ Now you have the [2D rendering context](/en-US/docs/Web/API/CanvasRenderingConte
 - Available rendering contexts: {{domxref("CanvasRenderingContext2D")}},
   {{domxref("WebGLRenderingContext")}} and {{domxref("WebGL2RenderingContext")}} and
   {{domxref("ImageBitmapRenderingContext")}}.
+- Available colorspaces: [sRGB color space](https://en.wikipedia.org/wiki/SRGB) and [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3)
```
Comment (wbamberg) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:195]:
```suggestion
- Available colorspaces: [sRGB color space](https://en.wikipedia.org/wiki/SRGB) and [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3).
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @dawei-wang !

**Pull request merged**